src/Connection.php

```diff
@@ -198,6 +203,8 @@ public function ping(): void
     /** @inheritdoc */
     public function disconnect()
     {
+        $this->connection?->removeSubscriber($this->commandSubscriber);
+        $this->commandSubscriber = null;
```

Comment:
Unregistering the subscriber seems correct, but I'm more generally curious how the `disconnect()` method is used in Laravel. Wouldn't it be misleading to users given that PHPC persists the connections by default?

I assume Laravel never uses the `disableClientPersistence` driver option.

Reply:
I'm just making sure that the objects are deleted without requiring a garbage collector cycle.

Reply:
I understand why you made the additions to this method. It was a more general question about what the purpose of `disconnect()` was for the Laravel MongoDB library. Is it something we're obligated to implement for Eloquent?